conda_build/build.py

```diff
@@ -855,6 +855,8 @@ def build(m, config, post=None, need_source_download=True, need_reparse_in_env=F
                     with path_prepended(config.build_prefix):
                         env = environ.get_dict(config=config, m=m)
                     env["CONDA_BUILD_STATE"] = "BUILD"
+                    if sys.platform != 'win32':
+                        env["CONDA_PATH_BACKUP"] = os.environ["CONDA_PATH_BACKUP"]
```

Comment:
Yeah since it was removed on conda master I'm hitting that issue. I was thinking to set s variable in conda_interface to the name of this env var actually? New version coming a little later..